R/processing.R

```diff
@@ -186,12 +186,13 @@ totalise_data <- function(data) {
 #' @importFrom tidyr drop_na
 #' @importFrom tidyselect all_of
 #' @importFrom rlang !! :=
-process_internal <- function(region, group_vars, totals = FALSE,
-                             localise = TRUE, verbose = TRUE) {
-  if (!any(class(region$clean) %in% "data.frame")) {
+process_internal <- function(clean_data, level, code, group_vars,
```

Comment:
looks good. Need to update documentation in line with this change and potentially testing (if `process_internal` is being tested outside of `process` which maybe it isn't.